ovos_plugin_manager/templates/tts.py

```diff
@@ -47,14 +46,17 @@
 EMPTY_PLAYBACK_QUEUE_TUPLE = (None, None, None, None, None)
 
 
-# TODO move to ovos_utils
-def get_ram_directory(folder):
-    tempfile = MemoryTempfile(fallback=True)
-    path = os.path.join(tempfile.gettempdir(), folder)
-    if not os.path.exists(path):
-        os.makedirs(path)
-    return path
-
+def get_temp_directory(folder):
+    if os.name == 'nt':
+        import tempfile
```

Comment:
this and the import below can remain at the top level of the module, doesnt need to be inside the function

Reply:
The import actually needs to be inside the function, as memory_tempfile fails to be imported in Windows.
I included the tempfile import inside the function for parity with memory_tempfile.
It is also possible to have a try importing memory_tempfile and make the code fall back to use tempfile, would you prefer this?

Reply:
if it throws an error in windows then it's fine to leave it like it is, if you can't tell I'm not a windows user and just assumed the import would work :P